dbms/src/Server/Server.cpp

```diff
@@ -1072,6 +1073,7 @@ int Server::main(const std::vector<std::string> & /*args*/)
       *  settings, available functions, data types, aggregate functions, databases...
       */
     global_context = Context::createGlobal();
+    global_context->setApplicationType(Context::ApplicationType::SERVER);
```

Comment:
This line should be set at global Context created. We missed to move this line in the previous PR.